src/external/modules/returns/lib/return-path.js

```diff
@@ -37,7 +37,7 @@ const isInternalEdit = (ret, request) => {
 const getEditButtonPath = (ret, request) => {
   if (isInternalEdit(ret, request)) {
     const returnId = getReturnId(ret);
-    return `/admin/return/internal?returnId=${returnId}`;
+    return `/return/internal?returnId=${returnId}`;
```

Comment:
I think this whole function may need removing from the external app - as external users shouldn't get an edit link.